tasks/update_appraisal_gemfiles.rake

```diff
@@ -22,7 +22,7 @@ task :install_appraisal_gemfiles do |_task, args|
   versions = tracer_version_arg.empty? ? TRACER_VERSIONS : tracer_version_arg
 
   versions.each do |version|
-    sh "docker-compose run -e APPRAISAL_GROUP --rm tracer-#{version} /bin/bash -c " \
+    sh "docker-compose run -e APPRAISAL_GROUP --rm tracer-#{version} --no-deps /bin/bash -c " \
```

Comment:
This ensures we don't start the whole docker-compose stack just to run `bundle`.

Reply:
Ahhh I totally did not know there was a shortcut for this. Nice! 👌